src/tests/ftest/control/dmg_system_reformat.py

```diff
@@ -58,30 +60,37 @@ def test_dmg_system_reformat(self):
             self.fail("Issues performing system erase: {}".format(
                 self.get_dmg_command().result.stderr_text))
 
-        # To verify that we are using the membership information instead of the
-        # dmg config explicit hostlist
-        # Uncomment below after DAOS-5979 is resolved
-        # self.assertTrue(
-        #     self.server_managers[-1].dmg.set_config_value("hostlist", None))
-
         self.log.info("Perform dmg storage format on all system ranks:")
-        self.get_dmg_command().storage_format(force=True)
-        if self.get_dmg_command().result.exit_status != 0:
-            self.fail("Issues performing storage format --force: {}".format(
-                self.get_dmg_command().result.stderr_text))
+
+        # Calling storage format after system stop too soon would fail, so
+        # wait 10 sec and retry up to 4 times.
+        count = 0
+        while count < 4:
+            try:
+                self.get_dmg_command().storage_format(force=True)
+                if self.get_dmg_command().result.exit_status != 0:
+                    self.fail(
+                        "Issues performing storage format --force: {}".format(
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string